.github/workflows/gen-binaries.yml

```
@@ -0,0 +1,40 @@
+name: Generate embedded binaries
+
+on:
+  push:
+    branches:
+      - main
```

Related to the previous comment we might want to trigger the workflow only when `nodetime` files are changed, maybe with:

```suggestion
on:
  push:
    branches:
      - main
    paths:
      - 'nodetime/*.go'
      - 'scripts/gen-nodetime'
      - 'scripts/data/gen-nodetime/**'
      - '!scripts/data/gen-nodetime/tsconfig.json'
```